From 2fab84f654cc1e120135426ad6fbabba866bd909 Mon Sep 17 00:00:00 2001
From: Herrie <Github.com@herrie.org>
Date: Fri, 10 May 2024 09:27:30 +0200
Subject: [PATCH] mm_vmscan.h: Update the kernel version check for
 mm_vmscan_lru_isolate

Seems that we have mm_vmscan_lru_isolate for Tissot, Mido and Sargo in 4.9 kernel due to https://github.com/shr-distribution/linux/commit/e6a14ff915c2cb0fb6f58d7d55efaa5e3b3f0fc5, even though this only hit in 4.11 in mainline kernel (https://github.com/torvalds/linux/commit/30b9aed8cd576964bff71a6c5f022ca30ac4c3b7).

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
Upstream-Status: Inappropriate [LuneOS Specific]

 include/instrumentation/events/mm_vmscan.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/instrumentation/events/mm_vmscan.h b/include/instrumentation/events/mm_vmscan.h
index ea6f4b78..6578f313 100644
--- a/include/instrumentation/events/mm_vmscan.h
+++ b/include/instrumentation/events/mm_vmscan.h
@@ -395,7 +395,7 @@ LTTNG_TRACEPOINT_EVENT(mm_vmscan_lru_isolate,
 		ctf_integer(int, lru, lru)
 	)
 )
-#elif (LTTNG_LINUX_VERSION_CODE >= LTTNG_KERNEL_VERSION(4,11,0))
+#elif (LTTNG_LINUX_VERSION_CODE >= LTTNG_KERNEL_VERSION(4,9,0))
 LTTNG_TRACEPOINT_EVENT(mm_vmscan_lru_isolate,
 
 	TP_PROTO(int classzone_idx,
